src/app/containers/InlineLink/index.jsx

```diff
@@ -25,8 +25,12 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
   if (result) {
     // the path is the first item in the array
     const path = result[0];
+    const hash = locator.split('#')[1] || null;
     return (
-      <InternalInlineLink to={path}>
+      <InternalInlineLink
+        to={{ pathname: path, hash, state: { hash } }}
+        aria-describedby={hash}
```

Comment:
What's the aria-describedby for please?

Reply:
Hi @greenc05. The `aria-describedby` attribute establishes a relationship between a link and the header element that it scrolls to for screen readers.

Reply:
I've gotten rid of this for now